.circleci/config.yml

```diff
@@ -349,50 +356,38 @@ workflows:
           requires:
             - checkout
 
-  react-next:
+  additional-tests:
     when:
-      equal: [react-next, << pipeline.parameters.workflow >>]
-    # triggers:
-    #   - schedule:
-    #       cron: '0 0 * * *'
-    #       filters:
-    #         branches:
-    #           only:
-    #             - master
+      and:
+        - equal: [additional, << pipeline.parameters.workflow >>]
```

Comment:
Not sure I follow this condition... There's no workflow called "additional", right?

Reply:
ahh, so the workflow isn't defined here, it's just being used when triggering the jobs 👍🏻